share/shutter/resources/modules/Shutter/App/Directories.pm

```diff
@@ -22,62 +23,56 @@
 
 package Shutter::App::Directories;
 
-use utf8;
+use 5.010;
 use strict;
 use warnings;
 
-#Glib
-use Glib qw/TRUE FALSE/;
+use Glib qw/ TRUE /;
 
-sub new {
-	my $class = shift;
+use constant {
+    SHUTTER_DIR        => "shutter",
+    UNSAVED_DIR        => "unsaved",
+    TEMP_DIR           => "temp",
+    AUTOSTART_DIR      => "autostart",
+    HIDDEN_SHUTTER_DIR => ".shutter",
+    PROFILES_DIR       => "profiles"
+};
 
-	my $self = {};
+sub create_if_not_exists : prototype($) {
```

Comment:
I thought prototypes aren't recommended generally, unless you're trying to mimic some perl builtins

Reply:
Usually yes, it's not worth to use them in the code, especially in OOP. But for small, often used functions it's OK. The main advantage of using prototypes is checking their invoking in compile time. If list of arguments in caller mismatches with prototype of the function, you will get an error and code won't run at all. But it works only with using `use` construction, not `require` and hence, now it isn't actual.
So, until `use Shutter::App::Directories` isn't used in the project, I agree that this is not needed here. I'll remove it

Reply:
ok, it's done in a6516a89